tools/prow-config-updater/updater.go

```diff
@@ -0,0 +1,189 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"knative.dev/pkg/test/cmd"
+	"knative.dev/pkg/test/ghutil"
+	"knative.dev/pkg/test/helpers"
+
+	"knative.dev/test-infra/shared/common"
+	"knative.dev/test-infra/tools/prow-config-updater/config"
+)
+
+type ConfigUpdater struct {
```

Comment:
This should be called `Client` in my opinion

Reply:
Changed to `Client`